src/platform/utilities/feature-toggles/featureFlagNames.json

```diff
@@ -43,6 +43,7 @@
   "covidVolunteerIntakeBackendEnabled": "covid_volunteer_intake_backend_enabled",
   "covidVolunteerIntakeEnabled": "covid_volunteer_intake_enabled",
   "covidVolunteerUpdateEnabled": "covid_volunteer_update_enabled",
+  "cstClaimPhase": "cst_claim_phase",
```

Comment:
Shouldn't this be `cst_claim_phases`?

Reply:
My bad yes let me update the ticket/details